pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -701,11 +759,20 @@ def cli(ctx, server, connection_name, default_namespace, user, password,
                                                 resolved_pull_max_cnt,
                                                 resolved_verify,
                                                 certfile, keyfile,
-                                                resolved_ca_certs,
+                                                ca_certs,
                                                 resolved_mock_server,
                                                 connections_repo,
-                                                conditional_options,
-                                                verbose)
+                                                verbose,
+                                                # used for test in function
+                                                default_namespace,
+                                                use_pull, pull_max_cnt, timeout,
+                                                verify)
```

Comment:
Similarly, this is a pain when debugging.

Reply:
DONE